lib/common/formats.js

```diff
@@ -740,6 +741,8 @@ declare const ${moduleName}: ${JSON.stringify(treeWalker(dictionary.tokens), nul
       allProperties = [...dictionary.allProperties].sort(sortByName);
     }
 
+    options = setComposeObjectProperties(options);
```

Comment:
Is it possible to choose the imports based on which transforms are being used instead of accepting arbitrary inputs in the configuration?

My concern is that by putting the inputs in the configuration we make it relatively easy to accidentally generate a Kotlin file that won't compile, whereas if we do the detection of which imports we actually need we can guarantee that the Kotlin file will compile correctly.

Reply:
I actually followed the same line as was already done with swift classes, sometimes custom import are needed